builder/tests/builder/src/test/java/io/helidon/builder/test/ComplexCaseTest.java

```diff
@@ -22,30 +22,45 @@
 import java.util.Map;
 import java.util.Set;
 
-import io.helidon.builder.test.testsubjects.ComplexCaseImpl;
+import io.helidon.builder.test.testsubjects.ComplexCase;
 import io.helidon.builder.test.testsubjects.MyConfigBean;
 
 import org.junit.jupiter.api.Test;
 
-import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+import static org.hamcrest.collection.IsMapContaining.hasEntry;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 class ComplexCaseTest {
 
     @Test
     void testIt() {
```

Comment:
we should still have at least a single toString() validation somewhere

Reply:
There are a few tests that still use toStrign (such as `CustomNamedTest`, `MyDerivedConfigBeanTest`)
What we should do is to have a dedicated test that only validates `toString()`, and never use it in any other test

Reply:
Created a follow up for this